target/linux/realtek/dts-5.10/rtl83xx_d-link_dgs-1210_gpio.dtsi

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+// Includes d-link dgs-1210 family common board functions connected to the rtl8231 GPIO expander, no PoE functions.
```

Comment:
try to keep comments 80 columns aligned ...

Reply:
Comment has been removed.